### ISSUE TITLE
Confusion between "Back" arrow icon (that cancels) and "Back" button (that goes back to the previous page)

### ISSUE THREAD

**Issue by ltfschoen** [paritytech/fether#380]
When I am navigating through a transaction. It often shows a "Back" button at the bottom. If I click it, it goes back to the previous page.  

There is also the arrow icon at the top left of the screen that is pointing backwards, and on most pages it just goes back to the previous page. But after navigating through a few pages to send a transaction, if you then click it, it cancels the whole process (taking you back multiple pages). For example, if I haven't send a transaction for a while, and I enter a transaction to a recipient for an amount, and click Scan, then scan the QR code with Parity Signer, then click Next Step, and then press the backward pointing arrow shown below, it takes me all the way back to my Account page

<img width="356" alt="screen shot 2019-01-22 at 7 33 54 am" src="https://user-images.githubusercontent.com/6226175/51498922-70173f00-1e1c-11e9-9661-3867138927fc.png">

On pages where the backward arrow is being used, if clicking it doesn't take the user to the previous page, could we please consider using a different icon that implies a "Cancellation" action instead of "Back" action, maybe a cross "X" shaped icon?

**Comment by Tbaut** [comment]
Yes, that's a good one. I'd suggest to keep the "back" behaviour for this arrow (this is used on mobile like this) and either use a "cancel" button, or do not use it at all.

+1 for the spacing.
